wrangling.Rmd

```diff
@@ -1026,23 +1012,18 @@ output <- data |>
   select(new_col)
 ```
 
-You can think of the pipe as a physical pipe. It takes the output from the
-function on the left-hand side of the pipe, and passes it as the first argument
-to the function on the right-hand side of the pipe. Note here that we have again
-split the code across multiple lines for readability; R is fine with this, since
-it knows that a line ending in a pipe `|>` is continued on the next line.
-Similarly, you see that after the first pipe, the remaining lines are indented
-until the end of the pipeline. This is not required for the R code to work, but
-again is used to aid in improving code readability.
-
-Next, let's learn about the details of using the pipe, and look at some examples
-of how to use it in data analysis.
-
-> **Note:** 
-> In this textbook, we will be using the base R pipe operator syntax, `|>`.
+> **Note:** You might also have noticed that we split the function calls across
+> lines after the pipe, similar to when we did this earlier in the chapter
+> for long function calls. Again this is allowed and recommended, especially when
+> the piped function calls create a long line of code. Doing this makes
+> your code more readable. When you do this, it is important to end each line
+> with the pipe operator `|>` to tell R that your code is continuing onto the
+> next line.
+
+> **Note:** In this textbook, we will be using the base R pipe operator syntax, `|>`.
 > This base R `|>` pipe operator was inspired by a previous version of the pipe
 > operator, `%>%`. The `%>%` pipe operator is not built into R 
-> and is from the `magrittr` R package.
+> and is from the `magrittr` R package. \index{pipesymbb@\texttt{%>%}|see{pipe}}
```

Comment:
index displays here: <img width="493" alt="Screen Shot 2021-09-29 at 5 00 25 PM" src="https://user-images.githubusercontent.com/3838091/135364454-b5bffb6e-7f90-4d38-9211-d1d1fd06ce13.png">

Reply:
moved to PDF output style guide